cmd/nexctl/main.go

```diff
@@ -73,6 +73,11 @@ func main() {
 					return nil
 				},
 			},
+			{
+				Name:   "connections",
+				Usage:  "Run a test of the nexd peer connectivity (host firewalls may block the ICMP probes)",
```

Comment:
Just a thought - it would be nice if we could query the keepalive status from wireguard instead of doing our own probes that are potentially dropped, but I don't see a way to do it.

Reply:
Agree! I noodled on this a bit. I * think * we could using interface rx/tx tracking. Another upside would be if the endpoint has a host based firewall it could be blocking ICMP. The downside is we are inferring that if counters are incrementing bidirectionally things are working. I think its definitely worth a back log to revisit.